scripts/update-chat.sh

```diff
@@ -7,7 +7,8 @@ set -x
 # directory with a GIT_VERSION.
 
 REPO_URL="https://github.com/posit-dev/py-shiny.git"
-BRANCH=main
+# TODO: Change this to main after this gets merged https://github.com/posit-dev/py-shiny/pull/1782
+BRANCH=markdown-stream-component
```

Comment:
Once https://github.com/posit-dev/py-shiny/pull/1782 gets merged

```suggestion
BRANCH=main
```